2048Games/20481

```diff
@@ -0,0 +1,2 @@
+tfe
```

Comment:
you should probably add these game data directories to your `.gitignore` so that they don't get checked in